src/Log/LogService.php

```diff
@@ -431,7 +427,7 @@ public function getDefaultFormat()
      * @param $format string
      * @returns string
      */
-    public function replaceFormatPlaceholder($format)
+    public function prepareFormat($format)
```

Comment:
This function can be public. It won't hurt the class. However it could also be protected.

Reply:
Just a comment. No action needed.